JSDemos/Demos/DataGrid/ColumnChooser/Angular/app/app.component.html

```diff
@@ -1,9 +1,10 @@
 <dx-data-grid
   id="employees"
   [dataSource]="employees"
-  keyExpr="ID"
+  [keyExpr]="'ID'"
```

Comment:
what's the difference?

Reply:
There is no difference at all.

But, in some projects component's input values are always set through square brackets.
It helps to differ it from the usual html attributes because you can add to an HTML element or component an HTML attribute too :)

In this case, I changed it to make the look of the DataGrid inputs consistent.
From my point of view, it looks better when all inputs have one style.